dbt/models/iasworld/schema/iasworld.dweldat.yml

```diff
@@ -1263,10 +1263,23 @@ sources:
             tests:
               - not_null:
                   name: iasworld_dweldat_char_attic_fnsh_not_null
-                  additional_select_columns: *select-columns
-                  config: *unique-conditions
+                  additional_select_columns:
+                    - parid
+                    - taxyr
+                    - card
+                    - who
+                    - wen
+                    - attic
+                  config:
+                    where: |
+                      CAST(taxyr AS int) BETWEEN {{ var('test_qc_year_start') }} AND {{ var('test_qc_year_end') }}
+                      AND cur = 'Y'
+                      AND deactivat IS NULL
+                      AND class NOT IN ('201', '213', '218', '219', '220', '221', '224', '225', '236', '240', '241', '290', '294', '297')
+                      AND attic != '3'
                   meta:
-                    description: user6 (Attic Finish) should not be null
+                    description: >
+                      user6 (Attic Finish) should not be null unless attic is 3 (None)
```

Comment:
It just occurred to me that I don't know if the reverse is true (should `user6` _always_ be null if attic is 3?) but I will check with Valuations. They're busy right now though, so I don't think the result of that discussion should block this PR.

Reply:
Almost certainly yes, but let's check with them to be sure.